Tiltfile

```diff
@@ -1,12 +1,17 @@
 # -*- mode: Python -*-
 
+envsubst_cmd = "./hack/tools/bin/envsubst"
+kubectl_cmd = "/usr/bin/kubectl"
+
 load("ext://uibutton", "cmd_button", "text_input")
 
 # set defaults
 version_settings(True, ">=0.22.2")
 
 settings = {
     "deploy_cert_manager": True,
+    "kubernetes_version": "v1.22.6",
+    "namespace": "default",
```

Comment:
Should this be the current namespace? also, what if the user provides another namespace in the kustomize_substitutions?

Reply:
I believe in this case it just sets a default value and will be overwritten by whatever's in `kustomize_substitutions`. In general, wdyt about setting default values for substitutions that we expect to see like `namespace` that can be overridden?